project-plan.md

```diff
@@ -82,3 +125,5 @@ Regular project deadlines:
 * 17-06-2016: BEP infosheet
 * 17-06-2016: SIG 2nd submission
 * 24-06-2016: Final presentation
+
+# Quality assurance
```

Comment:
I think you're right, deviating from the example report makes sense for this section.